content/concept-xrp-ledger-intro.md

```diff
@@ -1,43 +1,52 @@
 - **[On-Ledger Decentralized Exchange][]:** In addition to all the features that make XRP useful on its own, the XRP Ledger also has a fully-functional accounting system for tracking and trading obligations denominated in any way users want, and an exchange built into the protocol. The XRP Ledger can settle long, cross-currency payment paths and exchanges of multiple currencies in atomic transactions, bridging gaps of trust with XRP.
 
 ## Censorship-Resistant Transaction Processing
-[Censorship-resistant transaction processing]: #censorship-resistant-transaction-processing
+[Censorship-Resistant Transaction Processing]: #censorship-resistant-transaction-processing
 
-With (physical) coins and paper money, individuals can do business without going through a central party. Non-cryptocurrency digital assets all have a central administrator to confirm and validate transactions, who also has the power to censor or roll back transactions, or disallow some individuals from using the digital asset. (For example, if a digital money company decides a person has violated its terms of service, it can freeze or even confiscate that person's money.) XRP and other cryptocurrencies don't have a central administrator, so no one can roll back transactions, freeze XRP balances, or block someone from using them.
+XRP is part of a new class of money which includes Bitcoin and other cryptocurrencies:
 
-**Note:** Non-XRP currencies issued in the XRP Ledger _can_ be frozen. For more information, see the [Freeze documentation](concept-freeze.html).
+- These **Decentralized digital assets** exist in computer systems without a central administrator. As long as the system is sufficiently decentralized, no one can roll back transactions, freeze balances, or block someone from using a decentralized digital asset. These assets are natively digital, so they can be used online across any distance.
+
+This combines qualities of physical and centralized digital money. Prior to the invention of Bitcoin in 2009, all currencies could be divided into those two categories:
+
+- **Physical coins and paper money**, which individuals can use to do business without going through a central party. As physical objects, they cannot be used online, and doing business long-distance is slow and inconvenient.
+- **Centralized digital currencies**, which need an administrator to confirm transactions. The administrator also has the power to censor or roll back transactions, or disallow some individuals from using the digital currency. If company managing a digital currency decides someone has violated its terms of service, it can freeze or even confiscate that person's money. However, as digital balances, these currencies can be used online and are convenient across long distances.
```

Comment:
"If [the] company managing a digital currency" - maybe also change "company" to "operator", not all of these are run by companies.